expected/17.0/extra/returning.out

```diff
@@ -0,0 +1,519 @@
+---+---+------------+---+-------------------------+-------------------------
+ 1 | a | b          | c | 2017-11-06 12:34:56.789 | 2017-11-06 12:34:56.789
+(1 row)
+
+--Testcase 34:
+INSERT INTO "type_STRING"(col) VALUES ('string') ON CONFLICT DO NOTHING RETURNING *;
+  col   
+--------
+ string
+(1 row)
+
+--Testcase 35:
+ EXPLAIN (VERBOSE, COSTS OFF)
+INSERT INTO "type_STRING"(col) VALUES ('string') RETURNING *;
+           QUERY PLAN           
+--------------------------------
+ Insert on public."type_STRING"
+   Output: "type_STRING".col
+   Batch Size: 1
+   ->  Result
+         Output: 'string'::text
+(5 rows)
+
+--Testcase 36:
+INSERT INTO "type_BYTE"(col) VALUES ('c') RETURNING *;
+ col 
+-----
+ c
+(1 row)
+
+--Testcase 37:
+INSERT INTO "type_SINT"(col) VALUES (32767) RETURNING *;
+  col  
+-------
+ 32767
+(1 row)
+
+--Testcase 38:
+INSERT INTO "type_SINT"(col) VALUES (-32768) RETURNING *;
+  col   
+--------
+ -32768
+(1 row)
+
+--Testcase 39:
+INSERT INTO "type_BINT"(col) VALUES (9223372036854775807) RETURNING *;
+         col         
+---------------------
+ 9223372036854775807
+(1 row)
+
+--Testcase 40:
+INSERT INTO "type_BINT"(col) VALUES (-9223372036854775808) RETURNING *;
+         col          
+----------------------
+ -9223372036854775808
+(1 row)
+
+--Testcase 41:
+INSERT INTO "type_INTEGER"(col) VALUES (9223372036854775807) RETURNING *;
+         col         
+---------------------
+ 9223372036854775807
+(1 row)
+
+--Testcase 42:
+INSERT INTO "type_FLOAT"(col) VALUES (3.1415) RETURNING *;
+  col   
+--------
+ 3.1415
+(1 row)
+
+--Testcase 43:
+INSERT INTO "type_DOUBLE"(col) VALUES (3.14159265) RETURNING *;
+    col     
+------------
+ 3.14159265
+(1 row)
+
+--Testcase 44:
+INSERT INTO "type_TIMESTAMP" VALUES ('2017.11.06 12:34:56.789', '2017.11.06') RETURNING *;
+           col           |          b          
+-------------------------+---------------------
+ 2017-11-06 12:34:56.789 | 2017-11-06 00:00:00
+(1 row)
+
+--Testcase 45:
+INSERT INTO "type_TIMESTAMP" VALUES ('2017.11.06 1:3:0', '2017.11.07') RETURNING *;
+         col         |          b          
+---------------------+---------------------
+ 2017-11-06 01:03:00 | 2017-11-07 00:00:00
+(1 row)
+
+--Testcase 46:
+INSERT INTO "type_BLOB"(col) VALUES (bytea('\xDEADBEEF')) RETURNING *;
+    col     
+------------
+ \xdeadbeef
+(1 row)
+
+--Testcase 47:
+INSERT INTO typetest VALUES(1,'a', 'b', 'c','2017.11.06 12:34:56.789', '2017.11.06 12:34:56.789' ) RETURNING *;
+ i | v |     c      | t |            d            |           ti            
+---+---+------------+---+-------------------------+-------------------------
+ 1 | a | b          | c | 2017-11-06 12:34:56.789 | 2017-11-06 12:34:56.789
+(1 row)
+
+--Testcase 48:
+SELECT * FROM "type_DATE";
+    col     
+------------
+ 2021-02-23
+ 2021-03-08
+ 9999-12-30
+(3 rows)
+
+--Testcase 49:
+INSERT INTO "type_DATE"(col) VALUES ('2021.02.23') RETURNING col;
+ERROR:  Failed to execute remote SQL
```

Comment:
Could you add comment for the failed test cases below? What is your test purpose?